### PR TITLE
Pr split interface IMetricObserver

### PR DESCRIPTION
**Pull request by kbilsted**

### CODE REVIEW

**Comment by kbilsted** [comment]
What check is failing? I do not have access to see the error? All tests are green on my machine

**Comment by kichristensen** [comment]
@kbilsted I've restarted the build, as it failed while downloading some of the build tools.

File: src/Okanshi.InfluxDBObserver/InfluxDbObserver.cs [hunk 1]
```diff
@@ -9,8 +9,8 @@ namespace Okanshi.Observers
     /// <summary>
     /// Observer for posting metrics to InfluxDB.
     /// </summary>
-    public class InfluxDbObserver : IMetricObserver
-    {
+    public class InfluxDbObserver : IProcessingMetricObserver
```
Comment (kichristensen) [inline src/Okanshi.InfluxDBObserver/InfluxDbObserver.cs]: After this is splitted, there is no need for implementing GetObservations for the InfluxDbObserver

Comment (kbilsted) [reply src/Okanshi.InfluxDBObserver/InfluxDbObserver.cs]: fixed

**Pull request merged**